packages/cli/src/commands/generate-i18n.ts

```diff
@@ -0,0 +1,292 @@
+import dotenv from 'dotenv'
+dotenv.config({
+  path: [
+    path.resolve(process.cwd(), 'vtex.env'),
+    path.resolve(process.cwd(), '.env'),
+  ],
```

Comment:
suggestion: this is to support local development, right? What do you think about using only `.env`? This suggestion is because the store receives a warning of "`warning - Deprecated secret files detected: vtex.env`" when using the `vtex.env`, so its usage is discouraged, then I believe we don't need to worry about handling this file anymore.